test-data/unit/check-parameter-specification.test

```diff
@@ -2144,7 +2165,7 @@ from typing_extensions import ParamSpec
 P= ParamSpec("P")
 T = TypeVar("T")
 
-def smoke_testable(*args: P.args, **kwargs: P.kwargs) -> Callable[[Callable[P, T]], Type[T]]:
+def smoke_testable(*args: P.args, **kwargs: P.kwargs) -> Callable[[Callable[P, T]], Type[T]]:  # E: ParamSpec "P" is unbound
```

Comment:
this is functionality that mypy is losing, right? (you can see what it's going for / the OtherClass example is interesting)

i think that's fine (and seems fine looking at the new primer warnings), but maybe we should clarify this test case? e.g. the comments are out of date